README.md

```diff
@@ -172,6 +172,7 @@ Differences are noted here:
 |`safariLogAllCommunication`|Log all plists sent to and received from the Web Inspector, as plain text. For some operations this can be a lot of data, so it is recommended to be used only when necessary. Defaults to `false`.|`true` or `false`|
 |`safariLogAllCommunicationHexDump`|Log all communication sent to and received from the Web Inspector, as raw hex dump and printable characters. This logging is done _before_ any data manipulation, and so can elucidate some communication issues. Like `safariLogAllCommunication`, this can produce a lot of data in some cases, so it is recommended to be used only when necessary. Defaults to `false`.|`true` or `false`|
 |`safariSocketChunkSize`|The size, in _bytes_, of the data to be sent to the Web Inspector on iOS 11+ real devices. Some devices hang when sending large amounts of data to the Web Inspector, and breaking them into smaller parts can be helpful in those cases. Defaults to `16384` (also the maximum possible)|e.g., `1000`|
+|`safariWebInspectorMaxFrameLength`| The maximum size in bytes of a single data frame for the Web Inspector in integer. Defaults to 20MB (`20*1024*1024`) |e.g. `1024`, `100*1024*1024` |
```

Comment:
`in integer` could be dropped

Reply:
too high values could introduce slowness and/or memory leaks
too low values - possible buffer overflow exceptions